config.js

```diff
@@ -1567,6 +1567,8 @@ var config = {
     // You can enable tokenAuthUrlAutoRedirect which will detect that you have logged in successfully before
     // and will automatically redirect to the token service to get the token for the meeting.
     // tokenAuthUrlAutoRedirect: false
+    // An option to respect the context.tenant jwt field compared to the current tenant from the url
+    // tokenRespectTenant: false
```

Comment:
Add a trailing comma pl. also, how about we group token auth things in some tokenAuth section?

Reply:
Yeah, we can extract it at some point I don't think it is for this PR.